pylxd/image.py

```diff
@@ -84,3 +84,25 @@ def export(self):
         """Export the image."""
         response = self.api.export.get()
         return response.content
+
+    def add_alias(self, name, description):
+        """Add an alias to the image."""
+        self.client.api.images.aliases.post(json={
+            'description': description,
+            'target': self.fingerprint,
+            'name': name
+        })
+
+        # Update current aliases list
+        self.sync()
```

Comment:
Rather than call `sync` here, you should just add the alias to the object locally. Doing a sync will wipe away any local changes (which is a known issue), but it seems an awkward side effect that adding an alias loses local/unsync'd changes.